table/tables/cache_test.go

```diff
@@ -221,6 +221,7 @@ func TestCacheTableBasicReadAndWrite(t *testing.T) {
 		if tk.HasPlan("select * from write_tmp1", "UnionScan") {
 			break
 		}
+		time.Sleep(50 * time.Millisecond)
```

Comment:
Please add some comments on why adding the `Sleep` here.

Reply:
Done https://github.com/pingcap/tidb/pull/30924/files#diff-aa5d7f1cfcba5189e1a710fe7b29f53a8a5ef4ed9877af8e666fc66efe6db218R224

Reply:
Why wait for the cache to be loaded. I think when we got the UnionScan, the cache is already loaded.

Reply:
Did not see the break above... Ignore it...